build-managed.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 [platform] [useservergc]"
+    echo
+    echo "platform can be: FreeBSD, Linux, NetBSD, OSX, Windows."
+    echo "useservergc - Switch used by configure the hosted coreclr instance when executing tests."
+    echo
+}
+
+
+__scriptpath=$(cd "$(dirname "$0")"; pwd -P)
+
+__UnprocessedBuildArgs=
+__TestNugetRuntimeId=-distroRid
+__BuildOS=-os
+__TargetOS=-target-os
+__ServerGC=0
+
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    case $lowerI in
```

Comment:
If we do it in dir.props then these properties would be overridable, so if we decide to do it there, then we should just make sure that the values are correct.